pkg/sql/plan/function/builtin/debug/cmd_ping.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 - 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debug
+
+import (
+	"context"
+
+	"github.com/fagongzi/util/format"
+	"github.com/fagongzi/util/protoc"
+	"github.com/matrixorigin/matrixone/pkg/common/moerr"
+	pb "github.com/matrixorigin/matrixone/pkg/pb/debug"
+	"github.com/matrixorigin/matrixone/pkg/pb/metadata"
+	"github.com/matrixorigin/matrixone/pkg/pb/txn"
+	"github.com/matrixorigin/matrixone/pkg/vm/engine"
+)
+
+func handlePing(ctx context.Context,
+	serviceType serviceType,
```

Comment:
should this function be folded into caller? 

I believe almost all debug commands will do exactly the same thing other than different OpCode.    
Even payload could be rather generic -- but it if fine to require a pb struct.

Is it possible to make adding another function, for example echo, an almost one liner?